src/isar_exr/robotinterface.py

```diff
@@ -216,6 +216,8 @@ def initiate_mission(self, mission: Mission) -> None:
             mission_definition_id=mission_definition_id, robot_id=settings.ROBOT_EXR_ID
         )
 
+        time.sleep(5)
```

Comment:
Maybe add comment to why there is a sleep here?

Reply:
With more testing it seems like it is no longer neccessary after I increased the HTTP timeout, so I've removed it